### DISCUSSION TITLE
How to use named exports in next.js page component?

### DISCUSSION THREAD

**Discussion by tednaaa** [vercel/next.js#25940] [Help] [answered]
![image](https://user-images.githubusercontent.com/79831859/121362208-ffb6dd00-c946-11eb-93fb-06ea19045676.png)

how to use named export in page component like this?

I get error with that

![image](https://user-images.githubusercontent.com/79831859/121362495-2a089a80-c947-11eb-988e-bd39e09d39b8.png)

with export default all works finely

**Comment by akellbl4** [comment, marked as answer]
Hi @and-bulk.
It is no way to use named export for pages or templates like `_app.js` or `_documents.js` because Next.js doesn't know the names that you want to use. In this case, it seems reasonable to standardize exports and use only default exports. You can think about it as the API of Next.js.

In other cases such as your components or other files in your project, you can use named exports because you are a consumer of your code.

**Reply by tednaaa** [reply]
okay, Thanks for the support
